### PR TITLE
Update dependencies and make structure maven conform

### PR DESCRIPTION
**Pull request by Maxopoly**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -8,63 +8,55 @@
...
         <includes>
           <include>license.txt</include>
-        </includes>
-      </resource>
-      <resource>
-        <directory>${basedir}/src</directory>
-        <includes>
-          <include>**/*.java</include>
-          <include>*.yml</include>
-          <include>*.csv</include>
```
Comment (ProgrammerDan) [inline pom.xml:33]: Aren't these CSVs used by the built jar at runtime?

**Review by ProgrammerDan — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (Maxopoly) [reply pom.xml:33]: Arent they included automatically if moved into  src/main/resources?

Comment (ProgrammerDan) [reply pom.xml:33]: Just tested here: https://build.devotedmc.com/job/Bastion-pullrequest/ws/target/classes/

answer is no :(

Comment (Maxopoly) [reply pom.xml:33]: Ah okay, I thought that was the case. Will fix.

File: pom.xml [hunk 2]
```diff
@@ -8,63 +8,62 @@
...
           <include>license.txt</include>
+          <!--If a plugin has a decent README for once, let's include it -->
+          <include>README.md</include>
         </includes>
       </resource>
       <resource>
-        <directory>${basedir}/src</directory>
+        <directory>${basedir}/src/main/resources</directory>
         <includes>
-          <include>**/*.java</include>
```
Comment (Maxopoly) [inline pom.xml:31]: What was the idea behind this?

**Comment by Maxopoly** [comment]
Fixed

File: pom.xml (hunk 2 above)
Comment (ProgrammerDan) [reply pom.xml:31]: I think someone was trying to include the sources in with the distribution jar, check the license, might be a requirement.

Comment (Maxopoly) [reply pom.xml:31]: That one seems to be standard BSD 3

**Pull request merged**